### PR TITLE
Palettify editorial button colours

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

- Remove `format` prop from `EditorialButton`, `EditorialLinkButton` and `Toast` components
- Move `EditorialButton` colours into `palette`
- Update stories

## Why?

Defining colours in the palette lifts the colour styling decisions out of the component and up to the global scope. This means the component doesn't need to care about format per se, only the names of CSS custom properties whose values are set elsewhere.

### CODE REVIEW

File: dotcom-rendering/src/components/EditorialButton/EditorialButton.stories.tsx [hunk 1]
```diff
@@ -1,65 +1,18 @@
 import { css } from '@emotion/react';
```
Comment (domlander) [inline dotcom-rendering/src/components/EditorialButton/EditorialButton.stories.tsx:1]: When updating stories, we tend to update to CSF3: https://storybook.js.org/blog/storybook-csf3-is-here/

Some example PRs:
- https://github.com/guardian/dotcom-rendering/pull/12541
- https://github.com/guardian/dotcom-rendering/pull/12540
- https://github.com/guardian/dotcom-rendering/pull/12538

Not a blocker for this PR

File: dotcom-rendering/src/components/Toast.stories.tsx [hunk 2]
```diff
@@ -1,162 +1,63 @@
...
-					display: ArticleDisplay.Standard,
-				}}
-				count={3}
-				onClick={() => {}}
-			/>
-		</Wrapper>
+		<div
+			css={css`
+				position: relative;
+				height: 36px;
```
Comment (domlander) [inline dotcom-rendering/src/components/Toast.stories.tsx:25]:
```
import { space } from '@guardian/source/foundations';
height: ${space[9]}px;
```

**Review by domlander — APPROVED** [review]

**Comment by domlander** [comment]
The 12 Chromatic changes looks good to me, but I'll let you approve them in case you disagree.

File: dotcom-rendering/src/components/EditorialButton/EditorialButton.stories.tsx (hunk 1 above)
Comment (SiAdcock) [reply dotcom-rendering/src/components/EditorialButton/EditorialButton.stories.tsx:1]: Good shout! Done

**Pull request merged**